<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1100</width>
				<height>750</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>BOARD GAME PLATFORM - 棋类对弈平台</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<layout class="QHBoxLayout" name="horizontalLayout_main">
				<property name="spacing">
					<number>0</number>
				</property>
				<property name="leftMargin">
					<number>0</number>
				</property>
				<property name="topMargin">
					<number>0</number>
				</property>
				<property name="rightMargin">
					<number>0</number>
				</property>
				<property name="bottomMargin">
					<number>0</number>
				</property>
				<item>
					<!-- 左侧导航栏 -->
					<widget class="QFrame" name="sidebar">
						<property name="minimumSize">
							<size>
								<width>220</width>
								<height>0</height>
							</size>
						</property>
						<property name="maximumSize">
							<size>
								<width>220</width>
								<height>16777215</height>
							</size>
						</property>
						<property name="frameShape">
							<enum>QFrame::NoFrame</enum>
						</property>
						<layout class="QVBoxLayout" name="verticalLayout_sidebar">
							<property name="spacing">
								<number>5</number>
							</property>
							<property name="leftMargin">
								<number>0</number>
							</property>
							<property name="topMargin">
								<number>40</number>
							</property>
							<property name="rightMargin">
								<number>0</number>
							</property>
							<item>
								<widget class="QLabel" name="lblLogo">
									<property name="minimumSize">
										<size>
											<width>0</width>
											<height>60</height>
										</size>
									</property>
									<property name="text">
										<string>LOS CHESS</string>
									</property>
									<property name="alignment">
										<set>Qt::AlignCenter</set>
									</property>
									<property name="styleSheet">
										<string notr="true">font-size: 24px; font-weight: bold; color: white; letter-spacing: 2px;</string>
									</property>
								</widget>
							</item>
							<item>
								<spacer name="verticalSpacer_3">
									<property name="orientation">
										<enum>Qt::Vertical</enum>
									</property>
									<property name="sizeType">
										<enum>QSizePolicy::Fixed</enum>
									</property>
									<property name="sizeHint" stdset="0">
										<size>
											<width>20</width>
											<height>40</height>
										</size>
									</property>
								</spacer>
							</item>
							<item>
								<widget class="QPushButton" name="btnLobby">
									<property name="text">
										<string>  游戏大厅</string>
									</property>
									<property name="checkable">
										<bool>true</bool>
									</property>
									<property name="checked">
										<bool>true</bool>
									</property>
									<property name="class" stdset="0">
										<string>nav-btn</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="btnTeam">
									<property name="text">
										<string>  多人组队</string>
									</property>
									<property name="checkable">
										<bool>true</bool>
									</property>
									<property name="class" stdset="0">
										<string>nav-btn</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="btnProfile">
									<property name="text">
										<string>  个人档案</string>
									</property>
									<property name="checkable">
										<bool>true</bool>
									</property>
									<property name="class" stdset="0">
										<string>nav-btn</string>
									</property>
								</widget>
							</item>
							<item>
								<widget class="QPushButton" name="btnSettings">
									<property name="text">
										<string>  系统设置</string>
									</property>
									<property name="checkable">
										<bool>true</bool>
									</property>
									<property name="class" stdset="0">
										<string>nav-btn</string>
									</property>
								</widget>
							</item>
							<item>
								<spacer name="verticalSpacer">
									<property name="orientation">
										<enum>Qt::Vertical</enum>
									</property>
								</spacer>
							</item>
						</layout>
					</widget>
				</item>
				<item>
					<!-- 右侧主区域 -->
					<layout class="QVBoxLayout" name="verticalLayout_content">
						<property name="spacing">
							<number>0</number>
						</property>
						<item>
							<!-- 顶部 Header -->
							<widget class="QFrame" name="header">
								<property name="minimumSize">
									<size>
										<width>0</width>
										<height>80</height>
									</size>
								</property>
								<layout class="QHBoxLayout" name="horizontalLayout_header">
									<property name="leftMargin">
										<number>30</number>
									</property>
									<property name="rightMargin">
										<number>30</number>
									</property>
									<item>
										<widget class="QLabel" name="lblPageTitle">
											<property name="text">
												<string>GAMES LOBBY</string>
											</property>
											<property name="styleSheet">
												<string notr="true">font-size: 18px; color: #555; font-weight: bold;</string>
											</property>
										</widget>
									</item>
									<item>
										<spacer name="horizontalSpacer">
											<property name="orientation">
												<enum>Qt::Horizontal</enum>
											</property>
										</spacer>
									</item>
									<item>
										<widget class="QLabel" name="lblUserName">
											<property name="text">
												<string>Player_Guest</string>
											</property>
											<property name="alignment">
												<set>Qt::AlignRight|Qt::AlignVCenter</set>
											</property>
										</widget>
									</item>
									<item>
										<widget class="QLabel" name="lblUserAvatar">
											<property name="minimumSize">
												<size>
													<width>40</width>
													<height>40</height>
												</size>
											</property>
											<property name="maximumSize">
												<size>
													<width>40</width>
													<height>40</height>
												</size>
											</property>
											<property name="text">
												<string/>
											</property>
										</widget>
									</item>
								</layout>
							</widget>
						</item>
						<item>
							<!-- 游戏列表滚动区 -->
							<widget class="QScrollArea" name="scrollArea">
								<property name="widgetResizable">
									<bool>true</bool>
								</property>
								<widget class="QWidget" name="scrollAreaWidgetContents">
									<layout class="QVBoxLayout" name="verticalLayout_2">
										<property name="leftMargin">
											<number>30</number>
										</property>
										<property name="topMargin">
											<number>30</number>
										</property>
										<property name="rightMargin">
											<number>30</number>
										</property>
										<property name="bottomMargin">
											<number>30</number>
										</property>
										<item>
											<!-- 动态添加卡片的容器布局 -->
											<layout class="QGridLayout" name="gameLayout">
												<property name="horizontalSpacing">
													<number>20</number>
												</property>
												<property name="verticalSpacing">
													<number>20</number>
												</property>
											</layout>
										</item>
										<item>
											<spacer name="verticalSpacer_2">
												<property name="orientation">
													<enum>Qt::Vertical</enum>
												</property>
											</spacer>
										</item>
									</layout>
								</widget>
							</widget>
						</item>
					</layout>
				</item>
			</layout>
		</widget>
	</widget>
	<resources/>
	<connections/>
</ui>
